### ISSUE TITLE
Allow working with the app by using only the keyboard

### ISSUE THREAD

**Issue by introfog** [introfog/rGit#16]
Allow working with the app by using only the keyboard: Tabs, Esc and arrows.
Some functionality already implemented like tabs (by default from javafx, but probably tab order can be incorrect and should be fixed) and closing windows by clicking Esc (but not for the main window, because when you open directory chooser, and click Esc, it will automatically close directory chooser and main window, i still don't know how to fix it).
